region/crateria/west.json

```diff
@@ -1304,14 +1304,22 @@
                     "h_canNavigateUnderwater",
                     "canShinechargeMovement",
                     {"canComeInCharged": {
-                       "framesRemaining": 40,
-                       "fromNode": 2,
-                       "shinesparkFrames": 21,
-                       "excessShinesparkFrames": 7,
-                       "inRoomPath": [2,4]
-                     }}
+                      "framesRemaining": 40,
+                      "fromNode": 2,
+                      "shinesparkFrames": 21,
+                      "excessShinesparkFrames": 7,
+                      "inRoomPath": [2,4]
+                    }}
                   ],
                   "note": "Jump to either side and diagonal spark out."
+                },
+                {
+                  "name": "Underwater Walljumps",
+                  "notable": false,
+                  "requires": [
+                    "f_TourianOpen",
+                    "canUnderwaterWalljump"
+                  ]
```

Comment:
Curious on this one. Does the waterline matter?

Reply:
It doesn't but this does feel a little longer than the others